testsuite/forge/src/backend/k8s/swarm.rs

```diff
@@ -280,3 +298,100 @@ fn load_root_key(root_key_bytes: &[u8]) -> Ed25519PrivateKey {
 fn load_tc_key(tc_key_bytes: &[u8]) -> Ed25519PrivateKey {
     Ed25519PrivateKey::try_from(tc_key_bytes).unwrap()
 }
+
+fn clean_k8s_cluster_internal() {
```

Comment:
I think we do not need this fn wrapper, it only for Drop usage only + there is a pub fn below